packages/story-editor/src/components/library/panes/shopping/productDropdown.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useFeature } from 'flagged';
+
+/**
+ * External dependencies
+ */
+import { __ } from '@googleforcreators/i18n';
+import { useCallback, useEffect, useState } from '@googleforcreators/react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Datalist } from '@googleforcreators/design-system';
+import { useStory, useAPI } from '../../../../app';
+
+function ProductDropdown({ product, setProduct }) {
```

Comment:
Most of this code was moved here into a standalone component so it can be used in the floating menu and also the panel.

Reply:
Once we refactor the panel then it's only going to be used here, right? At that point we can move it closer to the floating menu component I suppose.

Reply:
Correct it's not used in #11392 - can make adjustments after this one goes in.